### PR TITLE
init fleet plugin install flagger

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
The KuratorCd feature requires the installation of a flagger for the nanomanaged cluster, which is implemented using the fleet plugin to install the flagger.

### CODE REVIEW

File: pkg/fleet-manager/manifests/plugins/flagger.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+type: oci
+repo: oci://ghcr.io/fluxcd/charts
+name: flagger
+version: 1.x # update Flagger to the latest minor version
+targetNamespace: flagger-system
+values:
+  nodeSelector:
+      kubernetes.io/os: linux
```
Comment (Xieql) [inline pkg/fleet-manager/manifests/plugins/flagger.yaml:8]: adding a blank line at the end to ensures file compatibility

Comment (Xieql) [inline pkg/fleet-manager/manifests/plugins/flagger.yaml:8]: adding a blank line at the end to ensures file compatibility

File: pkg/fleet-manager/plugin/plugin.go [hunk 2]
```diff
@@ -473,6 +475,36 @@ func buildStorageClusterValue(distributedStorageCfg fleetv1a1.DistributedStorage
 	return customValues
 }
 
+func RendeFlagger(
```
Comment (hzxuzhonghu) [inline pkg/fleet-manager/plugin/plugin.go:478]: can we add a test

**Review by hzxuzhonghu — APPROVED** [review]
Approve later

**Comment by LiZhenCheng9527** [comment]
fleet plugin install flagger has already been implemented and merged.

**Pull request closed without merging**